docker/test/stateful/Dockerfile

```diff
@@ -1,6 +1,7 @@
 # rebuild in #33610
 # docker build -t clickhouse/stateful-test .
 ARG FROM_TAG=latest
+# TODO consider replacing clickhouse with altinityinfra dockerhub account
```

Comment:
IIRC it is Ok not to replace NOW since we are not going to push images to dockerhub and only use it locally on machine it was build on. Am I right?

Reply:
Kind of. In this case, it's ok not to change right now because this action is only pulling. The other "actions" were pushing, because of that we needed to change.